modules/mongodb/testdata/v5.0.0/serverStatus.go

```diff
@@ -0,0 +1,2274 @@
+package v5_0_0
+
```

Comment:
It looks like we can remove `v500` prefix from all variables in this package.

`v5_0_0.V500ServerStatus` => `5_0_0.ServerStatus`

What do you think?

Reply:
done